File: CS/SalaryReports - Step 02/MainForm.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraReports.UI;

namespace SalaryReports
{
    public partial class MainForm : Form
    {
        public MainForm()
        {
            InitializeComponent();
        }

        private void simpleButton1_Click(object sender, EventArgs e)
        {
            EmployeeReport empReport = new EmployeeReport();

            ReportPrintTool printTool = new ReportPrintTool(empReport);
            printTool.ShowPreviewDialog();
        }
    }
}

File: VB/SalaryReports - Step 05/MainForm.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraReports.UI

Namespace SalaryReports
	Partial Public Class MainForm
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub simpleButton1_Click(ByVal sender As Object, ByVal e As EventArgs) Handles simpleButton1.Click
			Dim empReport As New EmployeeReport()

			Dim printTool As New ReportPrintTool(empReport)
			printTool.ShowPreviewDialog()
		End Sub
	End Class
End Namespace
